Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/pypa/manylinux2014_x86_64:2020-05-30-73c9395
+FROM quay.io/pypa/manylinux2014_x86_64
```

Comment:
Is there a tag available to ensure reproducible builds? If necessary we can probably set dependabot to automatically bump it.